### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by zkoch**

### CODE REVIEW

**Comment by stefania11** [comment]
LGTM

**Review by mdwelsh — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,21 +13,28 @@ you can create complex workflows with very little code.
...
 
-## Get your API key
+## Build Agents
+
+Agents are at the heart of the Fixie ecosystem, and we make it easy to build and contribute your own. To get started, check out our [examples repo](https://github.com/fixie-ai/fixie-examples) or [read the docs](https://docs.fixie.ai/agents/).
+
+## Using the Fixie CLI
 
-You can get your Fixie API key on your profile page at https://app.fixie.ai.
+In order to get started making requests to Fixie, you'll need an API key. You can get your Fixie API key on your profile page at https://app.fixie.ai.
```
Comment (hessamb) [inline README.md:27]: In order to get started making requests to Fixie, you'll need to authenticate. Simply:
```bash
fixie auth
```

File: README.md [hunk 2]
```diff
@@ -13,21 +13,28 @@ you can create complex workflows with very little code.
...
 ```
 $ export FIXIE_API_KEY=" -- your API key here -- "
 ```
 
-## Using the Fixie CLI
-
 The Fixie CLI tool lets you use Fixie interactively from the command line.
+
+Run `fixie --help` to see a list of available commands.
+
```
Comment (hessamb) [inline README.md:37]: The dict format below is deprecated. Now the output is in format:
```bash
➜  fixie console
Welcome to Fixie!
Connected to: https://app.fixie.ai/sessions/spark-typical-hardware
fixie 🚲❯ Hello there
   @user: Hello there
   @user: Hello there
   @fixie: Hello there
   @fixie: Thought: This is an atomic task.
   @fixie: Ask Agent: Hello there
   @router: Hello there
   @router: Sorry, I don't know an agent who can do that.
   @fixie: Agent says: Sorry, I couldn't reach the agent who can do that.
   @fixie: Thought: Router failed. I need to improvise but stick to facts.
   @fixie: Hi there! How can I help you?
1❯ Hi there! How can I help you?
```

File: README.md [hunk 3]
```diff
@@ -13,54 +13,63 @@ you can create complex workflows with very little code.
...
-fixie 🚲❯ Hello there
-   @{'handle': 'user'}: Hello there
-   @{'handle': 'user'}: Hello there
-   @{'handle': 'fixie'}: Hello there
-   @{'handle': 'fixie'}: Thought: This is not a question. I need to greet the user.
-   @{'handle': 'fixie'}: Hi there! How can I help you?
-1❯ Hi there! How can I help you?
-fixie 🚲❯
+Connected to: https://app.fixie.ai/sessions/spark-typical-hardware
+fixie 🚲❯  What's the price of Nvidia?
```
Comment (juberti) [inline README.md:37]:
```suggestion
fixie 🦊❯  What's the price of Nvidia?
```

**Review by juberti — APPROVED** [review]

**Pull request merged**